packages/v4/patternfly-docs/content/design-guidelines/components/wizard/wizard.md

```diff
@@ -5,45 +5,78 @@ section: components
 
-1. **Header (modal wizard only):** Modal wizards always have a header area that minimally contains the wizard title and a close button. The title should be action oriented and reflect the intended outcome of the wizard, such as "Create resource." You may optionally also include a description below the wizard title.
-2. **Steps sidebar:** Enumerated steps are displayed in the sidebar. Steps can be fixed or updated as the user proceeds through the process (see progressive wizard).
-3. **Visited step:** A step that has been already visited. In most cases users can click this step to return to a prior point in the flow.
-4. **Current step:** The current step is always highlighted.
-5. **Disabled step:** Steps that follow the current step are usually disabled to enforce the sequential flow.
-6. **Step title:** The Step title is unique to each step, and should reinforce the outcome of the individual step. It may match the Current step indicated in the steps sidebar, or as in the example image, it may be more verbose.
-7. **Body:** The contents of a step itself. Any valid form elements can be incorporated within the body. The size of the modal that contains the wizard can be adjusted based on the contents of the body section. If the height of the body section causes the bottom of the modal to push beyond the height of the viewport, a vertical scrollbar will appear. However it is recommended to break your workflow into small enough steps so that scrolling is not necessary on typical monitor sizes.
-8. **Button footer:** Buttons control the wizard flow. Default buttons are "Next," "Back," and "Cancel." You may optionally add other actions, such as "Skip to finish" or "Start over." There should only be one primary action in the footer and it should be the farthest left action.
+
+Step 1: The user is presented with a Get started screen where they can specify what they want to do.
+
+<img src="./img/wizard-progressive-step2-new.png" alt="Wizard progressive step 2" />
+
+
+Step 2: Based on their choice to create a new object, a user is presented with a second set of options. But the remaining steps are still unknown.
+
+<img src="./img/wizard-progressive-step3-new.png" alt="Wizard progressive step 3" />
+
+
+Step 3: After the user chooses “Quick create” and clicks "Next," they can now be presented with a full set of steps. It should be possible to revisit either of the first two steps making different choices and steps shown should update accordingly.
+
+### Modal wizards
+<img src="./img/modal-wizard.png" alt="Modal wizard" />
+
+The modal wizard lives in a modal dialog. If necessary, the default width and height of the modal can be overridden to increase that available content area. In most cases, the modal wizard is recommended as it will keep users focused on the task at hand. Users must either complete all of the steps of the wizard or cancel before navigating elsewhere within an application.
 
 #### Behavior
```

Comment:
Feel like this "Behavior" section should be an H2, and moved above the variations section

Reply:
Never mind, I now notice that there are multiple behavior sections... I wonder if it could be reworked to only have one big behavior section that combines all the behavior sections to explain how a wizard works (?)